docs/INSTALLATION.md

```diff
@@ -0,0 +1,11 @@
+# Windows Installation
+
+Powershell is included in most versions of windows
+
+# Linux Installation
+
+Download the appropriate RPM or DEB package from the [powershell releases page](https://github.com/PowerShell/PowerShell/releases/)
```

Comment:
Since we're talking about file formats, let's say `.rpm` and `.deb` here. Otherwise it gets complicated as RPM is the name of the package manager, but DEB is not.